lessons/Early-Math/evaluating-exponents/langs/en-us/pages/examples-exponents.adoc

```diff
@@ -0,0 +1,126 @@
+
+@hspace{2em} Test # @fitb{3em}{@ifsoln{1}} failed because the left side was @fitb{3em}{@ifsoln{25}} and the right side was @fitb{3em}{@ifsoln{32}}.
+
+@hspace{2em} Test # @fitb{3em}{@ifsoln{3}} failed because the left side was @fitb{3em}{@ifsoln{8}} and the right side was @fitb{3em}{@ifsoln{9}}.
+
+@hspace{2em} Test # @fitb{3em}{@ifsoln{5}} failed because the left side was @fitb{3em}{@ifsoln{243}} and the right side was @fitb{3em}{@ifsoln{15}}.
+
+@n Edit each of the *failing* examples so that _all examples pass_ when you click "Run". Be sure to change *only* the @ifproglang{wescheme}{second part of the example} @ifproglang{pyret}{part of the example after the `is`}! Describe one of the changes you had to make. @fitb{}{}
+
+@fitb{}{}
+
+@fitb{}{}
+
+@vspace{1ex}
+
+== Does it equal 16?
+
+@n A teacher asked her students to make up expressions with exponents that evaluate to 16. She typed their expressions into @proglang as examples to test if they evaluate to 16. Below, place a checkmark next to each of the examples that you *predict* will pass when you click "Run".
+
+@ifproglang{pyret}{
+
+[.table1, cols="1,12,12", frame="none", grid="none", stripes="none"]
+|===
+
+|| @fitb{1em}{} @hspace{1em} `num-expt(2, 4)` *is* `16`
+
+| @fitb{2em}{@ifsoln{x}} @hspace{1em} `2 * 2 * num-expt(2, 2)`  *is* `16`
+
+
+|| @fitb{2em}{@ifsoln{x}} @hspace{1em} `num-expt(2, 3) + 10` *is* `16`
+
+| @fitb{2em}{} @hspace{1em} `num-expt(4, 2) * 2`  *is* `16`
+
+|| @fitb{2em}{} @hspace{1em} `4 * num-expt(1, 4)`  *is* `16`
+
+| @fitb{2em}{} @hspace{1em}  `num-expt(4, 3) / 2 / 2`  *is* `16`
+
+|| @fitb{2em}{} @hspace{1em}  `2 * num-expt(2, 3)` *is* `16`
+
+| @fitb{2em}{@ifsoln{x}} @hspace{1em} `num-expt(4, 3) / 2 / 2`  *is* `16`
+|===
+
+}
+
+@ifproglang{wescheme}{
+
+[.table1, cols="1,12,12", frame="none", grid="none", stripes="none"]
+|===
+
+|| @fitb{1em}{} `(EXAMPLE (expt 2 4) 16)`
+
+| @fitb{2em}{@ifsoln{x}} `(EXAMPLE (* 2 (* 2 (* (expt 2 2)))) 16)`
+
+|| @fitb{2em}{@ifsoln{x}} `(EXAMPLE (+ (expt 2 3) 10) 16)`
+
+| @fitb{2em}{} `(EXAMPLE (* (expt 4 2) 2) 16)`
+
+|| @fitb{2em}{} `(EXAMPLE (* 4 (expt 1 4)) 16)`
+
+| @fitb{2em}{} `(EXAMPLE(/ (/ (expt 4 3) 2) 2) 16)`
+
+|| @fitb{2em}{}  `(EXAMPLE (* 2 (expt 2 3)) 16)`
+
+| @fitb{2em}{@ifsoln{x}}  `(EXAMPLE (/ (/ (expt 4 3) 2) 2) 16)`
+|===
+}
+
+@n Open the @starter-file{is-it-16} and click "Run". @ifproglang{pyret}{Select "Show Details" to the right of `examples-block-1`.} Which tests failed? @fitb{}{@ifsoln{Examples 2, 3, and 6.}}
+
+@n The students who wrote the three failing examples all have the same misconception about how to evaluate exponential expressions. What mistake did these three students make? @fitb{}{@ifsoln{Rather than multiplying the base times itself as many times as the exponent indicates,}}
```

Comment:
In principle, I _love_ this question! But I'm not confident that young students will understand the wording. Do they know what a "misconception" is?

I'm thinking maybe "The three failing examples are all wrong _for the same reason_. That's because the programmer who wrote them doesn't understand something about how exponents work! What do you they not understand?"